### PR TITLE
LSFF: add more model limitations; add some notes to model output section

### PR DESCRIPTION
**Pull request by NathanielBlairStahn**

### CODE REVIEW

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst [hunk 1]
```diff
@@ -2179,6 +2190,43 @@ Model Limitations
...
+  the impact of folic acid fortification.
+
+* GBD does not model folate deficiency, so we need to be careful that our
+  populations are comparable to the Keats study populations when applying the
+  effect size of folic acid fortification directly to the birth prevalence of
+  neural tube defects. (E.g., could there be some reason besides lack of
+  folate-rich food for a high rate of neural tube defects?)
+
+* The fortification scale-up strategy is relatively simple and does not capture
+  any of the
```
Comment (aflaxman) [inline docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst:2221]: any of the what??

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst [hunk 2]
```diff
@@ -2179,6 +2190,43 @@ Model Limitations
...
+    correlated with both flour consumption and fortification.
+
+  - Ideally, we would want to get correlations between birthweight, flour
+    consumption, and fortification in our populations, so that we could create a
+    joint distribution to appropriately sample all three variables at once.
+    Accounting for this correlation would likely reduce the impact from
+    increased birthweight.
+
+* GBD data for neural tube defects is too low, so we are likely underestimating
+  the impact of folic acid fortification.
```
Comment (alibow) [inline docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst:2212]: I think we should add more detail/information here. 
```suggestion
* GBD data for neural tube defects, particularly for India, is low relative to estimates of NTD burden from other data sources. Therefore, we are likely underestimating
  the impact of folic acid fortification in our simulation that relies upon GBD data.
```

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst (hunk 1 above)
Comment (NathanielBlairStahn) [reply docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: Oops, forgot to finish this. Added now.

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst (hunk 2 above)
Comment (NathanielBlairStahn) [reply docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: Thanks for the suggestion. I made this change and added citations.

**Pull request merged**